Cargo.toml

```diff
@@ -172,7 +172,7 @@ remain = "0.2.13"
 reqwest = { version = "0.12.2", default-features = false, features = ["rustls-tls", "json", "multipart"] }
 ring = "=0.17.5" # Upgrading this is possible, but a pain, so we don't want to pick up every new minor version (see: https://github.com/facebook/buck2/commit/91af40b66960d003067c3d241595fb53d1e636c8)
 rust-s3 = { version = "0.34.0-rc4", default-features = false, features = ["tokio-rustls-tls"] }
-rustls = { version = "0.22.3" }
+rustls = { version = "0.23.18", default-features = false}
```

Comment:
Was there a reason to exclude the default features? I think that's not going to work for the rest of the codebase

Reply:
For example, Buck2's [Cargo.toml](https://github.com/facebook/buck2/blob/a5dc441ea44e2eb3198b8d604fb7b2e65f6e3a1b/Cargo.toml#L251C1-L251C18) doesn't exclude the default features

Reply:
We discussed offline, but the short version is that `aws-lc-sys` (required by `aws-lc-rs`) has C/C++ code to compile and we want to use `ring`.

Reply:
Perfect, I understand now--we wanted to avoid the `aws-lc-rs` dep!